src/main/ipc.ts

```diff
@@ -102,6 +103,16 @@ const initialiseIpcHandlers: (ipcContext: IpcContext) => void = (
     requestTranscription(project)
   );
 
+  ipcMain.handle(
+    'set-confidence-lines-enabled',
+    async (_event, menuItemEnabled, confidenceLinesShown) =>
+      setConfidenceLinesEnabled(
+        ipcContext,
+        menuItemEnabled,
+        confidenceLinesShown
+      )
+  );
+
```

Comment:
`setConfidenceLinesEnabled` takes 2 arguments but here its given 3. Looks like you just need to rerun `yarn gen`